README.md

```diff
@@ -51,7 +53,9 @@ Alternatively, you can pre-compile the eBPF program, and provide it to the `trac
 2. `make bpf DOCKER=1` to build in a Docker container which includes all development tooling.
 3. There is also a handy `make all` (and the `make all DOCKER=1` variant) which builds both the executable and the eBPF program.
 
-Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above. In this case, the full Docker image can be replaced by the lighter-weight `aquasec/tracee:slim` image (this is unavilable at the moment and will be added to the next release). This image cannot build the eBPF program on its own, and is meant to be used when you already compiled the eBPF program beforehand, for example by adding the following mount to the docker run command: `-v /path/to/tracee.bpf.1_2_3.4_5_6.o:/tmp/tracee/tracee.bpf.1_2_3.4_5_6.o`.
+Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above. In this case, the full Docker image can be replaced by the lighter-weight `aquasec/tracee:slim` image (this is unavailable at the moment and will be added to the next release). This image cannot build the eBPF program on its own, and is meant to be used when you have already compiled the eBPF program beforehand.
+
+If running in a Docker container, regardless if it's the full or slim image, it's advisable to reuse the eBPF program across runs by mounting it from the host to the container. To make things easy, you can mount the entire `/tmp/tracee` directory into the container, so that if the container builds the eBPF program is will be persisted on the host, and if the eBPF program already exists on the host, the container can automatically discover it, for example `-v /tmp/tracee:/tmp/tracee`. This practice is helpful also to examine the output directory that Tracee creates under this path.
```

Comment:
"you can mount the entire `/tmp/tracee` directory...."
Maybe add "(or any other directory)"?

Reply:
"is will be persisted on the host" -> it ...
or "it will persist on the host"